.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
This file should be removed

Reply:
reverted in https://github.com/bbcmicrobit/micropython/pull/692/commits/849eb3def05ba7abb5e58441cae572f16509d6cd